targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Source/ExampleProject/ACLoudScriptTestResultUploader.h.ejs

```diff
@@ -0,0 +1,27 @@
+<%- copyright %>
+
+#pragma once
+
+#include "GameFramework/Actor.h"
+
+#include "PlayFab.h"
+#include "PlayFabCpp/Public/Core/PlayFabClientDataModels.h"
+#include "PlayFabCpp/Public/Core/PlayFabError.h"
+
+#include "ACloudScriptTestResultUploader.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogTestResults, Log, All); 
+
+UCLASS()
+class ACloudScriptTestResultUploader : public AActor
+{
+    GENERATED_BODY()
+public:
+    void UploadToCloudscript(TArray<class UPlayFabTestContext*> SuiteTests);
```

Comment:
I don't see anything inside the body of this function that should prevent it from being const.